feature/amazon/src/customEntitlementComputationDebug/kotlin/com/revenuecat/purchases/amazon/AmazonConfiguration.kt

```diff
@@ -0,0 +1,19 @@
+package com.revenuecat.purchases.amazon
+
+import android.content.Context
+import com.revenuecat.purchases.PurchasesConfiguration
+import com.revenuecat.purchases.Store
+
+class AmazonConfiguration(builder: Builder) : PurchasesConfiguration(builder) {
```

Comment:
Also had to create amazon classes for each flavor... Not sure if it's needed but adding it for now

Reply:
I'd be more inclined to keep Amazon only in the main flavor, unless there's a technical limitation that forces us to have it

Reply:
Hmm I thought of a way to do that, but will do it in a separate PR. For now, I'm just failing when using this in customEntitlementComputation mode.

Reply:
Done in https://github.com/RevenueCat/purchases-android/pull/1158